Automattic/wp-calypso
# Home: Fetch cards from API

#### Changes proposed in this Pull Request

Adds data layer methods for retrieving cards data from the `/wpcom/v2/sites/:site/home/cards` API endpoint developed in D40630-code.

#### Testing instructions

* ~~Apply D40630-code and sandbox the API.~~ (deployed)
* Go to the "My Home" section.
* Inspect the `Home` component with the React dev tools.
* Make sure it contains a `cards` prop with the data returned by the API.
<img width="1277" alt="Screen Shot 2020-03-24 at 13 37 11" src="https://user-images.githubusercontent.com/1233880/77426386-a9f32500-6dd4-11ea-81cf-0f3e9ec9f16d.png">
